lib/orchestrator.js

```diff
@@ -16,6 +16,9 @@ var TokenChecker = require('./token-checker');
 // Configuration file
 require('../config.js');
 
+// Pseudo-constants:
+var W3_ORG_PREFIX = /^https?:\/\/(www\.)?w3c?\.org/i;
```

Comment:
I think I'm just applying the *robustness principle* here. You may argue that accepting `w3c.org` and `https` is stretching it too much, but I don't see good reasons not to be liberal with the input, as long as it doesn't pose a security risk. `/TR` will be HTTPS some day, and many users don't bother typing `www`. This regex acknowledges that. If all other pieces of the software followed this principle (flexible, case-insensitive regexes to filter inputs where that is reasonable) maybe we would have less issues like #190.